src/staging/src/pages/active-advisories.js

```diff
@@ -161,10 +161,10 @@ const PublicActiveAdvisoriesPage = ({ data }) => {
 
     // exclude unpublished parks
     let q =
-      "/public-advisories/count?protectedAreas.published_at_null=false&protectedAreas.isDisplayed=true"
+      "/public-advisories/count"
 
     if (advisoryType !== "all") {
-      q += `&eventType.eventType_contains=${advisoryType}`
+      q += `?protectedAreas.published_at_null=false&protectedAreas.isDisplayed=true&eventType.eventType_contains=${advisoryType}`
```

Comment:
I think this line should be reverted based on our new understanding of the underlying issue.  Then I'll approve it with option 1 from my comment below.